LibationWinForms/GridEntry.cs

```diff
@@ -200,5 +260,12 @@ private static string GetMiscDisplay(LibraryBook libraryBook)
 		}
 
 		#endregion
+
+		~GridEntry()
+		{
```

Comment:
I've meant to bring it up before. I'm warry on destructors because GC can be such an unpredictable beast. It looks like you have more experience with them in practice though. How do you feel these compare to more explicit things like Dispose or OnClose?

Reply:
I wouldn't say I'm that experienced with them, but I think a finalizer is better used here than making GridEntry Disposable because:
1) There shouldn't be a problem with those event handlers if GridEntry lives longer than expected. They'll still fire, but they won't do anything since GridEntry has been unbound from the view.
2) The creator of a Disposable object is responsible for disposing of it, so ProductsGrid or Form1 would be responsible for iterating through the bindings list and manually disposing of every GridEntry each time the grid is reloaded.